src/components/CartItem.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteItem, incrementQuant, decrementQuant } from '../redux/coderSlice';
+
+
+
+const CartItem = () => {
+    const dispatch = useDispatch();
+    const productData = useSelector((state) => state.coder.productData);
+    return (
+        <div>
+            <div>
+                <h2>Cart Review</h2>
+            </div>
+            <div >
+                {
+                    productData.map((item) => (
+                        <div key={item.name}>
+                            <div className='cartitem-row'>
+                                <div onClick={() => dispatch(deleteItem(item.name))} className='cartitem-delete'>delete</div>
+                                <img className='cartitem-image' src={item.image} alt="" />
+                                <p>{item.name}</p>
+                                <p>${item.price}</p>
+                                <div>
+                                    <button onClick={() => dispatch(decrementQuant({
+                                        name: item.name,
```

Comment:
The object being passed in the `decrementQuant` and `incrementQuant` seems to be redundant. Can we create a function that generates this object?